src/test/java/com/iota/iri/controllers/TransactionViewModelTest.java

```diff
@@ -102,7 +111,14 @@ public void getApprovers() throws Exception {
 
     @Test
     public void fromHash() throws Exception {
+        TransactionViewModel tx = TransactionTestUtils.createBundleHead(0);
+        tx.store(tangle, snapshot);
+        Hash hash = tx.getHash();
+
+        PowerMockito.mockStatic(TransactionViewModel.class);
+        Mockito.when(TransactionViewModel.fromHash(tangle,hash)).thenReturn(tx);
 
+        assertEquals("Transaction hashes should be equal", hash, TransactionViewModel.fromHash(tangle, hash).getHash());
     }
```

Comment:
I don't like that one adds cod in a PR that has no relation to the current task at hand.
This is because:
1. The more code you add to the PR, the more potential problems you may add, and the harder it is to get the PR to pass
2. When you look through history and want to understand what was added and why you may get quite confused.

Next time create a separate PR. This time I will look the other way :)